### PR TITLE
Changed getActiveUniforms to return sequence of GLboolean for UNIFORM…

### PR DESCRIPTION
**Pull request by kenrussell**
…_IS_ROW_MAJOR.

This change is based on feedback from Janusz Sobczak from Mobica.

### CODE REVIEW

**Comment by kenrussell** [comment]
@zhenyao @grorg @jdashg please review.

@jasomobica FYI.

Note that UNIFORM_NAME_LENGTH is elided from this table, per the section on String Length Queries.

File: specs/latest/2.0/index.html [hunk 1]
```diff
@@ -2391,12 +2391,24 @@ <h2 class="no-toc">Table of contents</h2>
...
             <span class="gl-spec">(<a href="http://www.khronos.org/registry/gles/specs/3.0/es_spec_3.0.3.pdf#nameddest=section-2.11.6">OpenGL ES 3.0.3 &sect;2.11.6</a>,
             <a class="nonnormative" href="https://www.khronos.org/opengles/sdk/docs/man3/html/glGetActiveUniformsiv.xhtml">man page</a>)</span>
         <dd>
             Queries the value of the parameter named <code class="param">pname</code> for each of the uniforms within <code class="param">program</code> whose indices
-            are specified in the array of <code class="param">uniformIndices</code>.
+            are specified in the array of <code class="param">uniformIndices</code>. The type returned is the natural type for the requested
+            <code class="param">pname</code>, as given in the following table:
+            <table class="foo">
+                <tr><th>pname</th><th>returned type</th></tr>
+                <tr><td>UNIFORM_TYPE</td><td>sequence&lt;GLint&gt;</td></tr>
```
Comment (zhenyao) [inline specs/latest/2.0/index.html]: Should we change this to sequence<GLenum>?

Comment (kenrussell) [inline specs/latest/2.0/index.html]: Yes, thanks, good point. Will change this.

**Comment by kenrussell** [comment]
@zhenyao addressed your review feedback.

File: specs/latest/2.0/index.html (hunk 1 above)
Comment (jasomobica) [inline specs/latest/2.0/index.html]: I second that.

File: specs/latest/2.0/index.html [hunk 2]
```diff
@@ -2391,12 +2391,24 @@ <h2 class="no-toc">Table of contents</h2>
...
+                <tr><td>UNIFORM_SIZE</td><td>sequence&lt;GLint&gt;</td></tr>
```
Comment (jasomobica) [inline specs/latest/2.0/index.html]: And this could GLuint because negative values are not allowed for UNIFORM_SIZE

Comment (kenrussell) [inline specs/latest/2.0/index.html]: Will be changed in the next version.

**Comment by kenrussell** [comment]
Thanks for the feedback. Any further comments?

**Comment by zhenyao** [comment]
LGTM

**Pull request merged**